AngouriMath/Core/Sys/Entity/EntityProperties.cs

```diff
@@ -30,6 +30,13 @@ public EntityProperties(AngouriMath.Entity parent)
             return (T)properties[name];
         }
 
+        public bool GetPropIsFinite()
+        {
+            var res = GetProperty("IsFinite",
```

Comment:
This entire file should use `System.Lazy` instead of a type-unsafe Dictionary... The HasVariable one can use an `ImmutableSet<VariableEntity>` as the type argument of the `System.Lazy`.

Reply:
Better create an issue for it, I guess. Thanks